.gitattributes

```diff
@@ -4,3 +4,4 @@
 /.scrutinizer.yml   export-ignore
 /.travis.yml        export-ignore
 /docs               export-ignore
+/vagrant/provision/*.sh text eol=lf
```

Comment:
all sh should be forced to `lf` line ending otherwise provision script will fail after editing it under windows platform.